python_modules/dagster/dagster/_core/definitions/logical_version.py

```diff
@@ -390,6 +384,22 @@ def _gather_leaves(
             for child in cause.children:
                 yield from self._gather_leaves(child, level=level + 1)
 
+    def _convert_to_root_cause(self, cause: StaleStatusCause) -> StaleStatusRootCause:
+        if cause.reason == "updated dependency logical version":
+            assert cause.dependency, "[updated input] cause must have a dependency"
+            return StaleStatusRootCause(
+                cause.dependency,
+                "updated logical version",
```

Comment:
If we're making the shift to `data version` soon maybe we go ahead and change it here

Reply:
I have a stack of PRs that makes the "data version" rename but Nick hasn't given final blessing, there is still a question whether we'll use another name